modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/integration/KillCommandDdlIntegrationTest.java

```diff
@@ -0,0 +1,286 @@
+    }
+
+    /** */
+    @Test
+    public void testCancelComputeTask() {
+        CountDownLatch computeLatch = new CountDownLatch(1);
+
+        IgniteFuture<Collection<Integer>> fut = client.compute().broadcastAsync(() -> {
+            computeLatch.await();
+
+            return 1;
+        });
+
+        try {
+            IgniteUuid taskId = client.compute().activeTaskFutures().keySet().iterator().next();
+
+            executeSql(client, "KILL COMPUTE '" + taskId + "'");
+
+            assertThrowsWithCause(() -> fut.get(TIMEOUT), IgniteException.class);
+        } finally {
+            computeLatch.countDown();
+        }
+    }
+
+    /** */
+    @Test
+    public void testCancelTx() {
+        IgniteCache<Object, Object> cache = client.cache(DEFAULT_CACHE_NAME);
+
+        int testKey = PAGES_CNT * (PAGE_SZ + 1);
+
+        try (Transaction tx = client.transactions().txStart()) {
+            cache.put(testKey, 1);
+
+            executeSql(client, "KILL TRANSACTION '" + tx.xid() + "'");
+
+            assertThrowsWithCause(tx::commit, IgniteException.class);
+        }
+
+        assertNull(cache.get(testKey));
+    }
+
+    /** @throws Exception If failed. */
+    @Test
+    public void testCancelService() throws Exception {
+        String serviceName = "MY_SERVICE";
+
+        ServiceConfiguration scfg = new ServiceConfiguration();
+        scfg.setName(serviceName);
+        scfg.setMaxPerNodeCount(1);
+        scfg.setNodeFilter(grid(0).cluster().predicate());
+        scfg.setService(new TestServiceImpl());
+
+        client.services().deploy(scfg);
+
+        TestService svc = client.services().serviceProxy(serviceName, TestService.class, true);
+        assertNotNull(svc);
+
+        executeSql(client, "KILL SERVICE '" + serviceName + "'");
+        boolean res = waitForCondition(() -> grid(0).services().serviceDescriptors().isEmpty(), TIMEOUT);
+        assertTrue(res);
+    }
+
+    /** @throws Exception If failed. */
+    @Test
+    public void testCancelContinuousQuery() throws Exception {
+        IgniteCache<Object, Object> cache = client.cache(DEFAULT_CACHE_NAME);
+
+        ContinuousQuery<Integer, Integer> cq = new ContinuousQuery<>();
+
+        AtomicInteger cntr = new AtomicInteger();
+
+        cq.setInitialQuery(new ScanQuery<>());
+        cq.setTimeInterval(1_000L);
+        cq.setPageSize(PAGE_SZ);
+        cq.setLocalListener(events -> {
+            for (CacheEntryEvent<? extends Integer, ? extends Integer> e : events) {
+                assertNotNull(e);
+
+                cntr.incrementAndGet();
+            }
+        });
+
+        cache.query(cq);
+
+        for (int i = 0; i < PAGE_SZ * PAGE_SZ; i++)
+            cache.put(i, i);
+
+        boolean res = waitForCondition(() -> cntr.get() == PAGE_SZ * PAGE_SZ, TIMEOUT);
+        assertTrue(res);
+
+        Map<UUID, Object> routines = getFieldValue(grid(0).context().continuous(), "rmtInfos");
```

Comment:
Let's use continues queries system view instead of reflection, for example:
```
        SystemView<ContinuousQueryView> qrys = grid(0).context().systemView().view(CQ_SYS_VIEW);
        assertEquals(1, qrys.size());
        ContinuousQueryView view = qrys.iterator().next();

        UUID nodeId = view.nodeId();
        UUID routineId = view.routineId();
```

Reply:
System views are not available in CALCITE currently :(